overlord/snapshotstate/snapshotmgr.go

```diff
@@ -398,6 +399,12 @@ func doForget(task *state.Task, _ *tomb.Tomb) error {
 		return fmt.Errorf("internal error: task %s (%s) snapshot info is missing the filename", task.ID(), task.Kind())
 	}
 
+	if err := checkSnapshotConflict(st, snapshot.SetID, "import-snapshot",
```

Comment:
why this check here?

Reply:
Right, dropped it (related to the XXX comment below in Forget()). Thanks.